src/App.css

```diff
@@ -1,38 +1,68 @@
-.App {
-  text-align: center;
-}
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+
+.blue { /*blue divs*/
+  background-color: #1EBDF4;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 1em;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.grey { /*grey divs*/
+  background-color: #d3d3d3;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 1em;
```

Comment:
There's a lot of duplicated CSS here? It may be simpler to create one class that unites the common styles, so you only need to change the behaviour once.

Ex:
```css
.section {
  display: flex;
  flex-flow: row wrap;
  justify-content: center;
  padding: 1em;
}

.bg-blue {
  background-color: #1EBDF4;
} 

.bg-grey {
  background-color: #d3d3d3;
}
```